packages/web3-errors/package.json

```diff
@@ -0,0 +1,41 @@
+{
+	"name": "web3-errors",
+	"version": "1.0.0-alpha.0",
```

Comment:
also should be 4x. Isn't?

Reply:
When we create new packages even for 4.x, we start at 1.0.0 for [example](https://github.com/ChainSafe/web3.js/pull/4991/files/c57bb102daee6e7f0c619df0a7ab6b3433fccc84#diff-a6b43d54939f69b811f969bd81d65714cfbb7acf00b4d419d65d13b5e62f9a1bL29) the common package